pkgs/development/libraries/bamf/bamf-index.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/bamf-matcher.c b/src/bamf-matcher.c
+index 52dbd3a..b7167a4 100644
+--- a/src/bamf-matcher.c
++++ b/src/bamf-matcher.c
+@@ -1405,7 +1405,7 @@ fill_desktop_file_table (BamfMatcher * self,
+ 
+       bamf_matcher_add_new_monitored_directory (self, directory);
+ 
+-      bamf_file = g_build_filename (directory, BAMF_INDEX_NAME, NULL);
++      bamf_file = g_build_filename ("/run/current-system/sw/share/applications", BAMF_INDEX_NAME, NULL);
```

Comment:
Doesn't this break the package on other distros?

Reply:
I was to lazy to patch it to probably construct file names in $XDG_DATA_DIRS/applications/bamf.index.
Loading a bamf index won't work on other distros, but further down in the code it can load the table from a directory if the file doesn't exist.

Reply:
I also have no motivation to support bamf on non-nixos linux.